plugins/aws/tsup.config.ts

```diff
@@ -1,24 +1,11 @@
 import { defineConfig } from "tsup";
 
-export default defineConfig([
-  {
-    entry: ["src/index.ts", "src/sdk.ts"],
-    format: ["esm", "cjs"],
-    outDir: "dist",
-    dts: true,
-    banner: {
-      js: `import { createRequire } from 'module'; const require = createRequire(import.meta.url);`,
-    },
+export default defineConfig({
+  entry: ["src/index.ts", "sdk/index.ts", "lambda/index.ts"],
+  format: ["esm", "cjs"],
+  outDir: "dist",
+  dts: true,
+  banner: {
+    js: `import { createRequire } from 'module'; const require = createRequire(import.meta.url);`,
   },
```

Comment:
@gronxb 
I’m not sure why, but when I upload a zip file with this configuration, the Lambda doesn’t execute properly.

Reply:
but noExternal execute well.

Reply:
I think it should work the same way since moving a dependency from `dependencies` to `devDependencies` makes tsup automatically treat it as noExternal. But does it still not work?

Reply:
Lambda@Edge can only run in a CJS (CommonJS) environment, but if an index.js file exists, Lambda@Edge recognizes it as the default entry file.
The index.cjs file is executable since it follows CJS, but it seems that the banner added in tsup.config.ts is causing an error when executing it in CJS due to the import statement.

so I opened a new [PR](https://github.com/gronxb/hot-updater/pull/121).